compio-runtime/src/runtime/mod.rs

```diff
@@ -134,8 +134,31 @@ impl RuntimeInner {
             let res = std::panic::catch_unwind(AssertUnwindSafe(f));
             BufResult(Ok(0), res)
         });
-        // SAFETY: Just like spawn.
-        unsafe { self.spawn_unchecked(self.submit(op).map(|BufResult(_, op)| op.into_inner())) }
+        let closure = async move {
+            let mut op = op;
+            loop {
```

Comment:
Is it possible that we failed to create the notify handle due to some unrecoverable error?

Reply:
I don't think so. If dup fails, just waiting for some time should be OK.